CONTRIBUTING.md

```diff
@@ -1,297 +1,385 @@
+The metadata file should always include the package's current and historical maintainers. If you are editing a package, add yourself using a comma-delimited list format.
+If the user `chocolatey-community` is not already specified as the package's maintainer, this user must be added as the first maintainer in the `owners` element.
+One or more maintainers of the maintainers listed in the `owners` element should be equal to a similar name listed in our `CODE_OWNERS` file and will be considered the primary maintainer of the package.
 
-If the package needs to perform tasks that cannot be done with command line switches, e.g. clicking away a prompt during installation that cannot be suppressed like for example in the [dropbox](https://chocolatey.org/packages/dropbox) package, you can use [AutoHotkey](https://chocolatey.org/packages/autohotkey.portable). Make [autohotkey.portable](https://chocolatey.org/packages/autohotkey.portable) a dependency of the package. Prefer AutoHotkey over [AutoIt](https://chocolatey.org/packages/autoit.commandline), because AutoHotkey is more lightweight, FOS and more actively developed than AutoIt.
+#### Icons
 
-### 1.4.1 Work on all locales
+A package must have an icon available if the packaged software has an icon. This icon must be named the same as the package and is placed in the [icons](https://github.com/chocolatey-community/chocolatey-packages/tree/master/icons) directory.
+If the package name ends with either `.install` or `.portable`, the suffix may be ignored in the icon name.
+When an icon is added to this folder with the correct name, it will **automatically** be set in the metadata file and the Readme file when our build server updates the package, if the metadata file contains an `<IconUrl>` element.
 
-Script must work on every locale available for Windows, so be careful when using strings of text of window in the script that could be different in another language.
+**IMPORTANT: If no icon is available, the comment `<!-- IconUrl: Skip check -->` should be added to the metadata file**
 
-### 1.4.2 Avoid brittle scripts
+#### Chocolatey Compatibility
 
-Do not create brittle scripts that work only when user doesn't interfere. All script elements should be as precise as possible - for instance, instead of using [Send](https://autohotkey.com/docs/commands/Send.htm) function which will work correctly only if the desired window is active, use [ControlSend](https://autohotkey.com/docs/commands/ControlSend.htm) which doesn't require window activation or use [BlockInput](https://autohotkey.com/docs/commands/BlockInput.htm) for short period of time.
+All packages are expected to be compatible with the Chocolatey CLI version, available 12 months earlier than the last change of the package.
+That means if you change the metadata or one of the package scripts **on January 12th, 2023**, it should be compatible with Chocolatey CLI 0.11.3 and later released versions.
```

Comment:
Since we're now no longer supporting pre-1.0.0 versions, would it be worthwhile to update this date and version?

Reply:
As I have added a new Wiki Page for what we support, I'll update this section to link to that page.